<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SinifWidget</class>
 <widget class="QWidget" name="SinifWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="sinifEklemeButton">
     <property name="text">
      <string>Sinif Ekle</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="sinifSilmeButton">
     <property name="text">
      <string>Sinif Sil</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QListView" name="sinifList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
